operator/pkg/proxy/translation.go

```diff
@@ -52,7 +48,33 @@ func InjectProxyEnvVars(translator values.Translator, proxyEnv map[string]string
 			return nil, err
 		}
 
-		proxyVals, _ := getProxyConfigHelmValues(obj, proxyEnv) // ignore errors for now
-		return chartutil.CoalesceTables(vals, proxyVals), nil
+		proxyEnvVars, _ := getProxyConfigEnvVars(obj, proxyEnv) // ignore errors for now
```

Comment:
Um, what do you mean by "for now"? :confused:

Reply:
Valid question, given that the comment had been there for over a year now.

I've added an explicit log plus an explanation as to why we're not bubbling up the error.